diff -up cmake-3.1.0-rc1/Modules/CMakeNinjaFindMake.cmake.ninja cmake-3.1.0-rc1/Modules/CMakeNinjaFindMake.cmake
--- cmake-3.1.0-rc1/Modules/CMakeNinjaFindMake.cmake.ninja	2014-10-27 08:40:56.000000000 -0600
+++ cmake-3.1.0-rc1/Modules/CMakeNinjaFindMake.cmake	2014-10-29 19:36:04.758871982 -0600
@@ -12,6 +12,6 @@
 # (To distribute this file outside of CMake, substitute the full
 #  License text for the above reference.)
 
-find_program(CMAKE_MAKE_PROGRAM ninja
+find_program(CMAKE_MAKE_PROGRAM ninja-build
   DOC "Program used to build from build.ninja files.")
 mark_as_advanced(CMAKE_MAKE_PROGRAM)
diff -up cmake-3.1.0-rc1/Source/cmGlobalNinjaGenerator.cxx.ninja cmake-3.1.0-rc1/Source/cmGlobalNinjaGenerator.cxx
--- cmake-3.1.0-rc1/Source/cmGlobalNinjaGenerator.cxx.ninja	2014-10-29 19:36:04.761872039 -0600
+++ cmake-3.1.0-rc1/Source/cmGlobalNinjaGenerator.cxx	2014-10-29 19:37:00.528930970 -0600
@@ -1171,7 +1171,7 @@ std::string cmGlobalNinjaGenerator::ninj
              lgen->GetMakefile()->GetRequiredDefinition("CMAKE_MAKE_PROGRAM"),
                                     cmLocalGenerator::SHELL);
   }
-  return "ninja";
+  return "ninja-build";
 }
 
 std::string cmGlobalNinjaGenerator::ninjaVersion() const
